code/parachain/frame/fnft/src/test/nonfungibles/extrinsic.rs

```diff
@@ -0,0 +1,258 @@
+use std::collections::{BTreeMap, BTreeSet};
+
+use codec::Encode;
+use composable_tests_helpers::test::{block::process_and_progress_blocks, helper::RuntimeTrait};
+use composable_traits::{account_proxy::ProxyType, fnft::FinancialNft};
+use frame_support::{
+	assert_noop, assert_ok,
+	traits::tokens::nonfungibles::{Create, Inspect},
+};
+use sp_runtime::DispatchError;
+
+use crate::{
+	test::{
+		mock::{new_test_ext, Event, MockRuntime, Nft, Origin, Proxy},
+		prelude::{TEST_COLLECTION_ID, *},
+		ALICE, BOB, CHARLIE,
+	},
+	AccountIdOf, FinancialNftInstanceIdOf, Instance, OwnerInstances, Pallet,
+};
+
+/// Tests a simple transfer between 2 accounts, with only 1 total NFT existing.
+#[test]
+fn simple() {
```

Comment:
either rename this to `transfer_simple` (or something similar) or move it into a new module `transfer`